server/services/cohorts.js

```diff
@@ -78,6 +79,7 @@ const getPSICohorts = async (psiID) =>
     })
     .find({
       psi_id: psiID,
+      or: [{ 'participants.is_current': true }, { 'participants.is_current': null }],
```

Comment:
Is this a different bug that you're also fixing? This code hasn't been change recently has it?

Reply:
I'd say it's a different bug in a way. It selects participants who are no longer assigned to this cohort (`is_current` = false), hence the number of available participants is wrong.
This ticket says that the number of available participants is different in cohort table/cohort details. I'm trying to figure out why and I thought this could be one of the reasons

Reply:
Nevermind, Tavy says it's not how it should be - https://teams.microsoft.com/l/message/19:IUaAmNEwTGmcqt3CjlNCrc_j7pTI3rB5QRpVrTFxNHQ1@thread.tacv2/1663111279482?tenantId=5b973f99-77df-4beb-b27d-aa0c70b8482c&groupId=3f22a178-b2af-4d1a-99cd-bc1781c11210&parentMessageId=1663111279482&teamName=HCAP&channelName=General&createdTime=1663111279482&allowXTenantAccess=false
I'll update the query 👍